src/slices/errorSlice.js

```diff
@@ -2,20 +2,51 @@ export const errorReducer = (state = {}, action) => {
   const { type, payload } = action;
   const matches = /(.*)(Error)/.exec(type);
   // not an Error action, so we ignore them
+
+  // Get current time and date
+  var today = new Date();
+  var date =
+    today.getFullYear() +
+    "-" +
+    (today.getMonth() + 1) +
+    "-" +
+    today.getDate() +
+    ":" +
+    today.getTime();
+
```

Comment:
Koffor ikkje berre `Date.now()` ?

Reply:
Vess Date.now() returner **akkurat** det samme, så kan eg **kanskje** gå med på forslaget.

Reply:
"The Date.now() method returns the number of milliseconds elapsed since January 1, 1970 00:00:00 UTC." 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/now

Reply:
Formålet med date va vel å ha noke for å sjekke om staten e oppdatert? Treng ikkje å ha en formatert dato då

Reply:
Lettere å debug med formatert dato

Reply:
https://www.youtube.com/watch?v=1zbBzO2RdqM